src/pages/Login.tsx

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import login_image from '../assets/login-image.webp'
+import intercars_logo from '../assets/ic_new_logo.jpg'
+import { useForm, Controller, SubmitHandler } from "react-hook-form"
+import { IconButton } from '@mui/material';
+import InputAdornment from '@mui/material/InputAdornment';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+
+import { yupResolver } from "@hookform/resolvers/yup"
+import * as yup from "yup"
+
+const schema = yup
+    .object({
+        email: yup.string().required('Полето за имейл e задължително').email('Невалиден имейл адрес'),
```

Comment:
make sure to use `i18n` for translations. You can see an example inside `Projects.tsx`. 
In short you need to create a key value pairs for the different translations inside the file `/lang/bg.json` and then use the translations with the hook `useTranslation` and the key.

For example for the errors you can have something like this:
```
{
   errors: {
      email: {
          invalid: "Невалиден имейл адрес",
          required" "Полето за имейл е задалжилно"
      }
   }
}
```

and you can check this on how to translate the texts from yup https://stackoverflow.com/a/71194656/16586436

**You need to add translations to the whole page.**

You can use [this](https://github.com/vuestorefront/storefront-nuxt3-boilerplate/blob/develop/apps/web/lang/en.json) if you are unsure on how to format the json file.

Reply:
fixed